cpp/modmesh/view/RApplication.cpp

```diff
@@ -96,6 +101,39 @@ RApplication::RApplication(int & argc, char ** argv)
 
     menuBar->addMenu(appMenu);
 
+    auto * use_orbit_camera = new RAction(
+        QString("Use Oribt Camera Controller"),
+        QString("Use Oribt Camera Controller"),
+        [&]()
+        {
+            qDebug() << "Use Orbit Camera Controller";
+            auto * viewer = this->m_main->viewer();
+            viewer->scene()->setOrbitCameraController();
+            viewer->scene()->controller()->setCamera(viewer->camera());
+        });
+
+    auto * use_fps_camera = new RAction(
+        QString("Use First Person Camera Controller"),
+        QString("Use First Person Camera Controller"),
+        [&]()
+        {
+            qDebug() << "Use First Person Camera Controller";
+            auto * viewer = this->m_main->viewer();
+            viewer->scene()->setFirstPersonCameraController();
+            viewer->scene()->controller()->setCamera(viewer->camera());
+        });
+
+    auto * cameraGroup = new QActionGroup(this);
```

Comment:
We have been mixing camelCase and snake_name for local variables.  You didn't start it.  But we should start to think about how to end it.  It will probably take more time for us to accumulate code before coming up with a good guideline.